examples/speech_to_text/counter_in_tgt_lang_agent.py

```diff
@@ -14,13 +14,16 @@ class CounterInTargetLanguage(SpeechToTextAgent):
     def __init__(self, args):
         super().__init__(args)
         self.wait_seconds = args.wait_seconds
-        self.tgt_lang = args.tgt_lang
+        if args is not None:
+            with open(args.tgt_lang, "r") as file:
+                tgt_lang = file.read()
+        self.tgt_lang = tgt_lang
```

Comment:
The reason I added this part is just to preprocess the tgt_lang because we are reading it from a file. I came to this conclusion after several trials and errors. I noticed that before adding the logic, the evaluation works well but the tgt_lang is not being passed.

Reply:
One thing I'd also need clarification on is that in the counter_in_tgt_lang_agent, the tgt_lang is being passed to the class after inheriting from the parent class Agent, which means that the tgt_lang is not implemented in the parent class. I suppose the right thing will be to implement it in the parent class so that all the children classes will automatically have it

Reply:
We don't want to set tgt_lang in the init method by the end of this effort, so don't worry about moving this attribute to the parent class for now.

If the objective is still not clear to you, imagine that https://github.com/facebookresearch/SimulEval/blob/c7a1749e6e328f77a163b0d41c4104f5d3e5637d/examples/speech_to_text/counter_in_tgt_lang_agent.py#L22-L24 and https://github.com/facebookresearch/SimulEval/blob/c7a1749e6e328f77a163b0d41c4104f5d3e5637d/examples/speech_to_text/counter_in_tgt_lang_agent.py#L17 will be removed.  We want to be able to get the tgt_lang passed down to the `policy` method as part of  the `states`, something like:
```
def policy(self, states: Optional[AgentStates] = None):
    ....
    ....
    tgt_lang = states.tgt_lang
    if self.tgt_lang == "en": 
    ....
    ....
```

Reply:
Oh! Now I understand the goal. Thanks for the clarification!